clients/extension/scripts/popup.js

```diff
@@ -80,32 +82,36 @@ function find(query, context = document) {
 
 /**
  * @param {Error} err
- * @return {HTMLAnchorElement}
+ * @return {HTMLButtonElement}
  */
 function buildReportErrorLink(err) {
   const issueBody = `
 **Lighthouse Version**: ${getLighthouseVersion()}
 **Lighthouse Commit**: ${getLighthouseCommitHash()}
 **Chrome Version**: ${getChromeVersion()}
 **Initial URL**: ${siteURL}
-**Error Message**: ${err.message}
+**Error Message**: ${err.friendlyMessage || err.message}
```

Comment:
this will have to be ts-ignored or cast (we're saying it's an `Error` while `friendlyMessage` is only on `LighthouseError`)

Or we could pass in `message` as a string parameter since the caller has already figured that out.

Reply:
I still needed `err.stack` so I'm expecting a LighthouseError object.
Fixed in https://github.com/GoogleChrome/lighthouse/pull/8294/commits/84bac693c0ce0d722bd3bf2fa95838afcab258b2